spec/docs/npq_participants_spec.rb

```diff
@@ -56,6 +56,16 @@
     end
   end
 
+  it_behaves_like "JSON Participant Deferral documentation", "/api/v1/participants/npq/{id}/defer" do
+    let(:participant) { npq_application }
+    let(:attributes) do
+      {
+        reason: Participants::Defer::NPQ::REASONS.sample,
+        course_identifier: npq_application.npq_course.identifier,
+      }
+    end
+  end
```

Comment:
I don't think the npq endpoint should have the same schemas as the ecf ones - which it currently does with this shared examples setup.